src/e2e/specs/dashboard.spec.ts

```diff
@@ -31,8 +32,67 @@ test.describe("Breaches Dashboard - Headers", () => {
     expect(await dashboardPage.dashboardNavButton.getAttribute("href")).toEqual(
       "/redesign/user/dashboard",
     );
-    expect(await dashboardPage.fAQsNavButton.getAttribute("href")).toEqual(
+    expect(await dashboardPage.FAQsNavButton.getAttribute("href")).toEqual(
       "https://support.mozilla.org/kb/firefox-monitor-faq",
     );
   });
+
+  test("Verify that the site header and navigation bar is displayed correctly", async ({
+    dashboardPage,
+  }) => {
+    // link to testrail
+    test.info().annotations.push({
+      type: "testrail",
+      description:
+        "https://testrail.stage.mozaws.net/index.php?/cases/view/2301511",
+    });
+
+    // verify the navigation bar left side elements
+    await expect(async () => {
+      expect(
+        await dashboardPage.fireFoxMonitorLogoImgButton.isVisible(),
+      ).toBeTruthy();
+      expect(await dashboardPage.dashboardNavButton.isVisible()).toBeTruthy();
+      expect(await dashboardPage.exposuresHeading.isVisible()).toBeTruthy();
+      expect(await dashboardPage.FAQsNavButton.isVisible()).toBeTruthy();
+    }).toPass();
+
+    // verify the site header elements
+    await expect(async () => {
+      expect(await dashboardPage.actionNeededTab.isVisible()).toBeTruthy();
+      expect(await dashboardPage.fixedTab.isVisible()).toBeTruthy();
+    }).toPass();
+  });
+});
+
+test.use({ storageState: "./e2e/storageState.json" });
```

Comment:
What does this do here?
I figured it'd be within a container `describe()` block or within a test. But now I'm curious if this `test.use()` is used within tests above or if it causes any issues if tests are run out of order semi-randomly.

Reply:
@pdehaan This essentially bypasses login